docs/quickstart/live-demo.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Live demo
+enableToc: false
+---
+
+# Before you start
+
+You will need an OpenAPI specification. For testing purposes, you can use one of the following sample OpenAPI specifications: 
```

Comment:
```suggestion
You will need an OpenAPI definition. For testing purposes, you can use one of the following sample OpenAPI definitions: 
```

@skadinna what do you think about this?

Reply:
I see what you mean here. There is the specification that the definitions are based on. I have updated the specification to definition where appropriate.